### DISCUSSION TITLE
Import from Mobaxterm

### DISCUSSION THREAD

**Discussion by mgilank** [isontheline/pro.webssh.net#1015] [Ideas]
Hi, first, greate apps!
using your apps in my ipad.
And in need to import bunch of ssh session from mobaxterm(windows) to Webssh.
Termius already haave this feature. 
Hope you Webssh can do this also, 
Thanks!

**Comment by isontheline** [comment]
Hi @mgilank 👋

Thank you so much for your support 🙏

It could be great to import Mobaxterm hosts :)

Have you some (not-sensitive) export dump?

**Reply by mgilank** [reply]
hi
sure.
heres the inside export file
https://paste.jetorbit.dev/cfaa79db-75dd-4389-a9eb-594eb562ac44

save with file extension  .mxtsessions

**Reply by isontheline** [reply]
Thank you @mgilank 🙏

I need to to some reverse engineering on the second part :
<img width="899" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/563c6bdb-3921-4096-a297-f44cc1308df0">

Do you need to only import HOST + GROUP + USERNAME?
Which authentication scheme do you use?

**Reply by mgilank** [reply]
hi
yes,
what need to import (most important) is : ip, user, port, the name of its session, and group.
check this out , i explain in this image.
https://imgmu.com/GItqAk34.jpg

other details, i think, not that important , it just styling

**Reply by isontheline** [reply]
Hi @mgilank 👋

Thank you a lot for your image and explanations 🙏

How many days could you give me to implement this import without leaving WebSSH out of your SSH client choice?

**Comment by mgilank** [comment]
Hi
  
Please take your time as long as you need
  

  
Thannks!
